go_modules/spec/dependabot/go_modules/file_updater/go_mod_updater_spec.rb

```diff
@@ -225,10 +225,10 @@
             # OpenAPIV2 has been renamed to openapiv2 in this version
             let(:dependency_version) { "v0.5.1" }
 
-            it "raises a DependencyFileNotResolvable error" do
+            it "does not raises a DependencyFileNotResolvable error" do
               error_class = Dependabot::DependencyFileNotResolvable
               expect { updater.updated_go_sum_content }.
-                to raise_error(error_class) do |error|
+                to_not raise_error(error_class) do |error|
                 expect(error.message).to include("googleapis/gnostic/OpenAPIv2")
               end
```

Comment:
Could change this spec to expect the update now that it is successful afaict:

```suggestion
              it "updates the go.mod" do
                expect(updater.updated_go_mod_content).to include(
                  %(github.com/googleapis/gnostic v0.5.1\n)
                )
              end
```

Reply:
Thank you, this makes more sense Philip 👍🏼